problem_builder/questionnaire.py

```diff
@@ -158,7 +158,14 @@ def author_edit_view(self, context):
         Add some HTML to the author view that allows authors to add choices and tips.
         """
         fragment = self.get_author_edit_view_fragment(context)
-        fragment.add_content(loader.render_template('templates/html/questionnaire_add_buttons.html', {}))
+
+        # Let the parent block determine whether to display review-related questionnaire options.
```

Comment:
Ah, makes more sense that way.

Addressed in [questionnaire.py](https://github.com/open-craft/problem-builder/pull/122/commits/a72c2836c7708191008e4755f2ca4a8dfdb5db9f#diff-3ac5402f0b62ec845cd52b31321b1528R824), and [mentoring.py](https://github.com/open-craft/problem-builder/pull/122/commits/a72c2836c7708191008e4755f2ca4a8dfdb5db9f#diff-88effd2bde10c4ba755bff423957d7a6R162).